src/cd.js

```diff
@@ -16,13 +16,17 @@ function _cd(options, dir) {
       dir = common.state.previousDir;
   }
 
-  if (!fs.existsSync(dir))
+  // This complexity is so that we only stat once.
+  try {
+    var stat = fs.statSync(dir);
+    if(stat.isDirectory()) {
+      common.state.previousDir = process.cwd();
+      process.chdir(dir);
+    } else {
+      common.error('not a directory: ' + dir);
+    }
+  } catch (e) {
```

Comment:
Just tested this out, and it looks like both `common.error()` statements get run if the user types something like `cd('file.txt')` (where `file.txt` is a regular file that exists). Could we fix this?